### ISSUE TITLE
401 Unauthorized error by ApiTokensController#show

### ISSUE THREAD

**Issue by mikesmayer** [catarse/catarse#2518]
i'm coming across a 401 Unauthorized error when running on my local machine

```
Started GET "/api_token" for ::1 at 2017-03-07 19:05:30 -0500
Processing by ApiTokensController#show as JSON
Completed 401 Unauthorized in 3ms (Views: 0.2ms | ActiveRecord: 0.0ms)
```

if I visit http://localhost:3004, i see this:

`[{"schema":"1","name":"banks","insertable":true},{"schema":"1","name":"categories","insertable":false},{"schema":"1","name":"category_totals","insertable":false},{"schema":"1","name":"contribution_activities","insertable":false},{"schema":"1","name":"contributors","insertable":false},{"schema":"1","name":"countries","insertable":false},{"schema":"1","name":"direct_messages","insertable":true},{"schema":"1","name":"finished_projects","insertable":false},{"schema":"1","name":"project_contributions","insertable":false},{"schema":"1","name":"project_contributions_per_day","insertable":false},{"schema":"1","name":"project_contributions_per_location","insertable":false},{"schema":"1","name":"project_contributions_per_ref","insertable":false},{"schema":"1","name":"project_details","insertable":false},{"schema":"1","name":"project_posts_details","insertable":false},{"schema":"1","name":"project_reports","insertable":true},{"schema":"1","name":"project_scores","insertable":false},{"schema":"1","name":"project_stat_contributors","insertable":false},{"schema":"1","name":"project_transitions","insertable":true},{"schema":"1","name":"projects","insertable":false},{"schema":"1","name":"reward_details","insertable":true},{"schema":"1","name":"states","insertable":false},{"schema":"1","name":"statistics","insertable":false},{"schema":"1","name":"successful_projects","insertable":false},{"schema":"1","name":"team_members","insertable":false},{"schema":"1","name":"team_totals","insertable":false},{"schema":"1","name":"user_details","insertable":false}]`

**Comment by mikesmayer** [comment]
I wish this was easier to debug -- the code seems to have thrown an error while I convert it to server :en translations

**Comment by sushant12** [comment]
@mikesmayer Have u saved jwt_secret token?

**Comment by mikesmayer** [comment]
@sushant12, yes -- this was a result of being only partially turned on :en within the app. technically I've resolved it, but the architecture of multi-language support needs to be cleaned up.

 think instead of :name_pt vs everything else, maybe we could adopt native plus translated versions
